translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -104,7 +104,7 @@ func resourceSpansToJaegerProto(rs pdata.ResourceSpans) (*model.Batch, error) {
 func resourceToJaegerProtoProcess(resource pdata.Resource) *model.Process {
 
 	process := model.Process{}
-	if resource.IsNil() {
+	if resource.IsEmpty() {
 		process.ServiceName = tracetranslator.ResourceNotSet
 		return &process
 	}
```

Comment:
I think this is also not necessary, it is not different from `if attrs.Len() == 0 {` case below.

Reply:
Done.